src/test/java/seedu/address/logic/commands/FindTagCommandTest.java

```diff
@@ -31,14 +40,19 @@ void equals() {
         TagContainsKeywordsPredicate secondPredicate =
                 new TagContainsKeywordsPredicate(Collections.singletonList("second"));
 
-        FindTagCommand findFirstCommand = new FindTagCommand(firstPredicate);
-        FindTagCommand findSecondCommand = new FindTagCommand(secondPredicate);
```

Comment:
Looks like you can also delete the predicates above 😄

Reply:
Thanks for spotting this! 😄